### DISCUSSION TITLE
API for all the GFX calls ?

### DISCUSSION THREAD

**Discussion by www-asics-ws** [moononournation/Arduino_GFX#92] [General]
Hi,

is there a description of all the graphic calls somewhere ?

Also, how do you manage fonts ?

Thanks,
rudi

**Comment by moononournation** [comment]
PDQGraphicTest example nearly demonstrated all functions except gfx fonts. And HelloWorldGFX example show the missing part.

**Reply by www-asics-ws** [reply]
Reverse engineering and searching through example files doesn't give me the min/max parameter values.
A small write-up about all commands would be very helpful ...

Thanks for the hard work !

**Reply by moononournation** [reply]
I agree it will be handy if have a good documentation, welcome for your contribution ;)

**Comment by www-asics-ws** [comment]
I wonder if you could help me with something else:
I need to share the SPI bus with other devices. My TFT screen is working now.

I am using:
Arduino_DataBus *bus = new Arduino_HWSPI(TFT_DC, TFT_CS, &SPI, true);

And then, I use:
SPI.transfer16(dac_out, SPI_LAST);

I manage the CS myself. I can see on m y scope that the chip selects are correct. But when I try to send out "dac_out" is not being sent on the SPI bus.

Any idea what I am doing wrong ?

Thank you !

rudi
